src/components/contestants.jsx

```diff
@@ -0,0 +1,207 @@
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <h2>Concurs Național de Informatică</h2>
+              </Col>
+            </Row>
+            <Row className="big-spacing" />
+          </Grid>
+        </div>
+        <Grid className="stats-section">
+          <Row>
+            <Col md={6} mdOffset={3}
+                 sm={8} smOffset={2}
+                 xs={12}>
+              <Row className="inner-stats">
+                <Col xs={4}>
+                    <p className="description">Participanți</p>
+                    <p className="value">75</p>
+                </Col>
+                <Col xs={4} className="border-left">
+                    <p className="description">Proiecte</p>
+                    <p className="value">39</p>
+                </Col>
+                <Col xs={4} className="border-left">
+                    <p className="description">Județe</p>
+                    <p className="value">17</p>
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+        </Grid>
+        <div className="white-section-wrapper">
+          <Grid>
+            <Row className="big-spacing" />
+            <Row className="white-section">
+              <Col mdOffset={2} md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="educational" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="media" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="robots" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="utility" />
+              </Col>
+              <Col md={1}>
+                <FilterIcon currentCategory={this.state.currentCategory}
+                            toggleCategory={this.toggleCategory}
+                            category="web" />
+              </Col>
+              <Col mdOffset={2} md={1}>
+                <Glyphicon glyph="th-large" className={gridClassName} onClick={this.showGrid} />
+              </Col>
+              <Col md={1}>
+                <Glyphicon glyph="align-justify" className={tableClassName} onClick={this.showTable} />
+              </Col>
+            </Row>
+          </Grid>
+          <Grid>
+              <Row className="small-spacing" />
+              {this.state.showGrid ?
+                <Row className="white-section projects-grid hidden-xs">
```

Comment:
Defapt scoate si cu if-ul de sus. Fa ca in [ultimul exemplu](https://facebook.github.io/react/tips/if-else-in-JSX.html).